test/test_initial_design/test_initial_design.py

```diff
@@ -119,3 +121,18 @@ def test_init_budget(self):
             'but provided none of them.',
         ):
             InitialDesign(**kwargs, n_configs_x_params=None)
+
+    def test__select_configurations(self):
```

Comment:
why __?

Reply:
Because the function is called _select_configurations